lib/Frontend/CompilerInvocation.cpp

```diff
@@ -459,8 +459,12 @@ static bool ParseSearchPathArgs(SearchPathOptions &Opts,
                            /*isSystem=*/A->getOption().getID() == OPT_Fsystem});
   }
 
+  for (const Arg *A : Args.filtered(OPT_vfsoverlay)) {
+    Opts.VFSOverlayFiles.push_back(resolveSearchPath(A->getValue()));
+  }
+
   for (const Arg *A : Args.filtered(OPT_L)) {
-    Opts.LibrarySearchPaths.push_back(resolveSearchPath(A->getValue()));
+    Opts.ImportSearchPaths.push_back(resolveSearchPath(A->getValue()));
```

Comment:
This change is incorrect; please revert. (-L paths go to the frontend for immediate mode, not for import resolution.)